core/storage/improvements/gae_models.py

```diff
@@ -26,7 +26,7 @@
 
 (base_models,) = models.Registry.import_models([models.NAMES.base_model])
 
-TASK_ENTITY_TYPE_EXPLORATION = feconf.ENTITY_TYPE_EXPLORATION
+TASK_ENTITY_TYPE_EXPLORATION = feconf.TASK_ENTITY_TYPE_EXPLORATION
```

Comment:
I don't get it -- why does this need to be in feconf at all?

Reply:
No, that's not the case. If you want to share constants use assets/constants.ts.

Reply:
Also, backend constants don't need to go in feconf by default. It's actually preferable to keep them in the relevant file otherwise feconf becomes a bit of a jumble. We only move them to feconf when there's a need to share them across multiple import chains.